### ISSUE TITLE
pager not work

### ISSUE THREAD

**Issue by terryfth** [aliyun/aliyun-cli#512]
* **Command Format**: 

* **Execution Command**: aliyun  vpc  DescribeVSwitches -o cols=VSwitchId rows=VSwitches.VSwitch[] --pager

* **Output**:
VSwitchId
---------
vsw-uf68ixh4vocrr1qsd1bxm
vsw-uf63ibjv1ck6yhama39xu
vsw-uf6mka4j8ic7bg427laxl
vsw-uf6xzhz3i60paeebyv6xe
vsw-uf6wdh1c54pd0f4sf1wxo
vsw-uf65y8xxn8zhd2u3lehxp
vsw-uf6m5josswvctx6tw5fxc
vsw-uf64dl7xxxje0lrcx1fuxh
vsw-uf6ndmcw9mqm373xwxj
vsw-uf6xu0a9im12mqa2ffdxh

--pager doesn't work, I have over 10+ vswitch, --pager not get them out, but --PageSize 50 does

**Comment by generals-space** [comment]
yeah, same like `ecs DescribeImages`, and `--PageSize` can not larger than 100.

```console
$ aliyun ecs DescribeImages --PageSize 101
ERROR: SDK.ServerError
ErrorCode: InvalidParameter
Recommend: https://next.api.aliyun.com/troubleshoot?q=InvalidParameter&product=Ecs
RequestId:
Message: The specified parameter "PageSize" is not valid.
RespHeaders: map[Access-Control-Allow-Origin:[*] Connection:[keep-alive] Content-Length:[278] Content-Type:[application/json;charset=utf-8] Date:[Mon, 10 Oct 2022 14:35:46 GMT] X-Acs-Request-Id:[] X-Acs-Trace-Id:[]]
```

But `--PageSize 100` works well, maybe you can add this to documents

**Comment by terryfth** [comment]
--pager 3.0.82 still works.  after 3.0.83, it fails

**Comment by zhangyoufu** [comment]
可以尝试 `--pager NextToken=` 作为 workaround
